### PR TITLE
feat: defillama api key input front

### PR DESCRIPTION
**Pull request by cleanerzkp**
need to be rebased with backend

Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Review by kelsos — APPROVED** [review]

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -2462,6 +2462,12 @@
       "hint": "Enter your CryptoCompare API key",
       "title": "CryptoCompare"
     },
+    "defillama": {
+      "delete_tooltip": "Delete your DefiLlama Pro API key",
+      "description": "Used to query higher rate limits for DeFi prices and data. This API key will increase your query limits for DeFi-related information.",
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:2467]:
```suggestion
      "description": "Used to query higher rate limits for DeFi prices and data. This API key will increase your query rate limits",
```

**Review by LefterisJP — COMMENTED** [review]
Tested it and it works fine. Left some smaller comment in the description.

And on top of that:

1. Add a link to the how toget the api key. https://defillama.com/pro-api
2. Edit the docs to show we also have the option of adding this key

**Review by LefterisJP — APPROVED** [review]
LGTM

**Review by lukicenturi — APPROVED** [review]
LGTM!

**Pull request merged**